hazelcast-client-new/src/test/java/com/hazelcast/client/spi/impl/discovery/ClientDiscoverySpiTest.java

```diff
@@ -179,10 +208,36 @@ public void testNodeFilter_from_xml() throws Exception {
         assertEquals(4, nodeFilter.getNodes().size());
     }
 
+    @Test
+    public void test_discovery_address_translator() throws Exception {
+        String xmlFileName = "hazelcast-client-discovery-spi-test.xml";
+        InputStream xmlResource = ClientDiscoverySpiTest.class.getClassLoader().getResourceAsStream(xmlFileName);
+        ClientConfig clientConfig = new XmlClientConfigBuilder(xmlResource).build();
+
+        ClientNetworkConfig networkConfig = clientConfig.getNetworkConfig();
+
+        DiscoveryStrategiesConfig discoveryStrategiesConfig = networkConfig.getDiscoveryStrategiesConfig();
+        assertNotNull(discoveryStrategiesConfig);
+        assertNotNull(discoveryStrategiesConfig.getNodeFilterClass());
+        ILogger logger = Logger.getLogger(ClientDiscoverySpiTest.class);
+
+        DiscoveryServiceProvider provider = new DefaultDiscoveryServiceProvider();
+        DiscoveryService discoveryService = provider.newDiscoveryService(null, logger, DiscoveryMode.Client,
+                discoveryStrategiesConfig, ClientDiscoverySpiTest.class.getClassLoader());
+
+        AddressTranslator translator = new DiscoveryAddressTranslator(discoveryService);
+
+        Address address = new Address("127.0.0.1", 50001);
+
+        translator.translate(null);
```

Comment:
I'm missing something here; but what is being tested? Should the return value of the translate method not be verified?